.ci_support/linux_64_.yaml

```diff
@@ -1,8 +1,10 @@
 cdt_name:
-- cos7
+- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

Comment:
This default value of `python_min` comes from the global [conda-forge-pinnning](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/1514d2938250bcdf37df42e728017ec0ecc40afb/recipe/conda_build_config.yaml#L849). It is overriden by the `{% set python_min = '3.10' %}` in `recipe/meta.yaml`